hosts_test.go

```diff
@@ -151,6 +151,51 @@ func TestFindHosts(t *testing.T) {
 
 }
 
+func TestFindHostByCustomIdentifier(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {
+		if req.URL.Path != "/api/v0/hosts-by-custom-identifier/mydb001" {
```

Comment:
`name` and `customIdentifier` are the same value. So, in this test code, it is ambiguous from which parameter the URL string is generated.

Reply:
`customIdentifier` can be any string. It is preferable to use `customIdentifier` that requires percent-encoding as a test case.